src/containers/TicTacToe/index.js

```diff
@@ -0,0 +1,40 @@
+import React, { useState } from 'react'
+import Button from '../../components/Button'
+import Board from '../Board';
+import { playMove, resetBoard } from '../../actions/restActions'
+import './index.css'
+
+
+function TicTacToe() {
+  const [symbol, setSymbol] = useState('x')
+  const [board, setBoard] = useState(["-","-","-","-","-","-","-","-","-"]);
+
+  return (
+    <div>
+      <div className="control-btns">
+        <Button 
+          mark="X" 
+          handleClick={() => {}} 
```

Comment:
It might be better to have this as the default behavior for `Button` component. So when you don't pass in an argument for `handleClick`, it either doesn't do anything or trigger an empty body callback.